### PR TITLE
Toolbar001

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
**Modified**:

- [ ] index file
- [ ] style file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,31 +12,31 @@
...
-                     I can help you build a product ,
+                     I can help you build a product
                       feature or website Look through
                        some of my work and experience!
                         If you like what you see and 
                         have a project you need coded,
                          don’t hestiate to contact me.
                  </p>
-                 <a href="##">LET’S CONNECT</a>
+                 <h1 class="a">LET’S CONNECT</h1>
```
Comment (ichala) [inline index.html]: - [ ] To be semantically correct , for best SEO performance & as mentioned by the previous reviewer, you should only have one `<h1>` tag on your page, kindly update `<h1 class="a">LET’S CONNECT</h1>`  to `<h2 class="a">LET’S CONNECT</h2>` 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @ismailmunyentwari9 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson


🛑🛑🛑 Note : please do not close or merge the PR until you get approved , try always to make the requested changes in the same PR

## To Highlight 🎉

- Most of the changes Implemented  ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ismailmunyentwari9) [reply index.html]: may you please review my changes please

Comment (ismailmunyentwari9) [reply index.html]: please check my changes I have already fixed h1 mistake  in my coddes

**Review by BregornOriginal — APPROVED** [review]
## Hi @ismailmunyentwari9 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳 🥳 🥳 

## Congratulations! 

giphy (2)

## Highlights

✅ Linters are passing
✅ Excelent README file

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


- [OPTIONAL] ***From now, start doing commits that's are meaningful. For example, instead of saying `added` or `fixed` you should say `add` or `fix`***

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**